src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/DevAppServerArgs.java

```diff
@@ -24,30 +24,42 @@
 public class DevAppServerArgs {
 
   /**
-   * @return [--name=value] or [] if value=null.
+   * @return {@code [--name=value]} or {@code []} if value=null.
```

Comment:
This is the norm in every other similar comment for this, and 3 other classes. I agree we should improve these comments, but how do you feel about doing this in another PR?